packages/code-du-travail-frontend/src/lib/useLocalStorage.ts

```diff
@@ -12,13 +12,11 @@ const initialValue = (key, defaultValue) => {
 
 const updateValueCallback = (key, setValue) => (value) => {
   setValue(value);
-  if (!window.localStorage) {
-    return;
-  }
+
   try {
-    window.localStorage.setItem(key, JSON.stringify(value));
+    window.localStorage?.setItem(key, JSON.stringify(value));
```

Comment:
pour fixer les erreurs :
`Failed to read the 'localStorage' property from 'Window': Access is denied for this document.`

Reply:
je fais le check dans le `try/catch` au lieu de le faire à l'exterieur